crowbar_framework/app/models/ntp_service.rb

```diff
@@ -74,6 +74,16 @@ def transition(inst, name, state)
   def apply_role_pre_chef_call(old_role, role, all_nodes)
     Rails.logger.debug("NTP apply_role_pre_chef_call: entering #{all_nodes.inspect}")
 
+    net_svc = NetworkService.new @logger
+    listen_networks = role.default_attributes["ntp"]["listen_networks"] || []
```

Comment:
Also rename the attribute here to `server_listen_on_networks`.

Reply:
Good catch, thanks !